tests/e2e/e2e_setup_test.go

```diff
@@ -170,6 +174,13 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	} else {
 		s.runIBCRelayer()
 	}
+
+	// Check if we should skip ibc tests
+	if skipEVMTests {
+		s.T().Log("skipping EVM tests e2e preparation")
+	} else {
+		s.setupEVMAccountOnChain(s.chainA, 0)
+	}
```

Comment:
Wrong comment on IBC. Making an account that long that we should skip it? Why not creating it all the time?